giskard_cicd/loaders/huggingface_inf_model.py

```diff
@@ -89,8 +68,15 @@ def predict_from_text_classification_inference(df: pd.DataFrame) -> np.ndarray:
                 sleep(0.5)
                 output = query(payload)
 
-            for i in output:
-                results.append(extract_scores(i))
+            try:
+                for i in output:   
+                    sorted_output = sorted(i, key=lambda x: labels.index(x["label"]))
+                    results.append([x["score"] for x in sorted_output])
+            except Exception as e:
```

Comment:
We will lose all outputs if one of them has problem.

Let's try to handle it in the loop and provide a default value for the missing prediction case